epsilons/main_online_experiments.py

```diff
@@ -28,19 +28,35 @@
 
 flags.DEFINE_integer("initial_seed", "1", "the program will run seeds [initial_seed, initial_seed + 5)")
 
+flags.DEFINE_string("experiment", "1", "the experiment will be run in")
```

Comment:
I don't understand what this flag does

Reply:
explain it a little more please

Reply:
This flag chooses which experiment to run. For example, epsilon, learning rate, and so on.

Reply:
it should be: 
flags.DEFINE_string("experiment", "epsilons", "the experiment will be run in")

Reply:
Then please update it